Assets/Source/Components/Player/PlayerBehavior.cs

```diff
@@ -59,17 +63,24 @@ public override void ComponentAwake()
 
             cameraEffector = GetRequiredComponent<CameraEffectComponent>(cameraObject);
             menuSelector = GetRequiredComponent<CanvasMenuSelectorComponent>(FindOrCreateCanvas());
-            levelDirector = GetRequiredComponent<DirectorComponent>(FindLevelObject());
 
-            actorDash = gameObject.AddComponent<ActorDash>();
-            actorDash.CooldownTime = DASH_COOLDOWN;
-            actorDash.DashDistance = DASH_DISTANCE;
+            actorDashBehavior = gameObject.AddComponent<ActorDashBehavior>();
```

Comment:
This should be added to the prefab, not added via code.  Sorry i missed this before.